frontend/src/components/chat/ChatBody.js

```diff
@@ -20,11 +20,11 @@ const ChatBody = ({ chatSocket, username, room_id, is_live }) => {
   })
 
   chatSocket.on('message response', (data) => {
-    console.log(data)
-    setMessages([...messages, data])
+    let newMessages = [...messages, data];
     if (is_live) {
-      axiosApiInstance.post(URL_USER_SVC_MESSAGE, {room_id: room_id, messages: messages})
+      axiosApiInstance.post(URL_USER_SVC_MESSAGE, {room_id: room_id, messages: newMessages})
```

Comment:
Nice, i happened to have written this in the branch "[origin/feat/room-page-layout](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g27/tree/origin/feat/room-page-layout)" for the currently open PR as well.